src/collective/fingerpointing/upgrades/configure.zcml

```diff
@@ -0,0 +1,29 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:gs="http://namespaces.zope.org/genericsetup">
+
+  <include file="profiles.zcml" />
+
+  <gs:registerProfile
```

Comment:
@hvelarde i used to do it this way but ended up with broken upgrade steps because the profile configurations change over time:

imagine someone has  installed collective.fingerpointing on his/her site a month ago. then does a bad job at maintining the site and directly upgrades to the most recent version in 2020.

collective.fingerpointing will have evolved in the meantime and about 20 upgrade steps have been added. 
the upgrade step introduced in this release will then not just add the new "view audit log" action to portal_actions/user but possible other actions that have been added as well.
the same for mapping of permissions in rolemap.xml (they might add - not yet registered- permissions to roles)
actions will be added multiple times, assertions that have been made when coding upgrade step logic all of a sudden won't be true any longer:

eg. a portal_type is not yet registered but added to `Products.CMFPlone.interfaces.controlpanel.INavigationSchema.plone.displayed_types`  in `registry.xml`. there is a validator for this schema field that will fail and stop the upgrade.


so the lesson i learned here is: it's not a good idea to simply run (selected) import steps of the package profile in an upgrade step because you don't know the content of their configuration file when the upgrade is run.